### DISCUSSION TITLE
Org support

### DISCUSSION THREAD

**Discussion by mankoff** [magit/forge#514] [Ideas]
It would be nice to use the power of Org for working with Magit Forge.

I don't see mention of Org in Discussions, but note that many Issues mention Org, but I think this is only because it's listed in the debug strings pasted by users. This suggests many Magit and Forge users are also Org users.

One feature could be bulk viewing and editing of Issues (tags, times, sizes, assignees, etc.) via Org column view.

Another feature could be viewing Issues (single or groups) as a standard Org file, using tags and properties, allowing edits that can be pushed back to the Forge. This is similar to how https://github.com/nyyManni/ejira presents Jira issues

It is already possible to edit issues in Org mode and have them converted to Markdown automagically.

Forge allows private notes (and other metadata for a topic?). It would be nice to support SCHEDULE and DEADLINE data privately, so that topics can appear in an Org Agenda view.

**Comment by mankoff** [comment]
Above I mentioned `viewing Issues (single or groups) as a standard Org file, using tags and properties, allowing edits`. Even without edits, if Topics could be presented as Org files, then the Org Agenda can be used to build custom views into the Topics - PRs vs Issues, categorizing and grouping by tag or status, etc. It lets Org Agenda be a Kanban viewer.

**Comment by tarsius** [comment]
> This suggests many Magit and Forge users are also Org users.

I don't remember "Org support" to be an often requested feature.

While I understand the appeal of "Org support", I don't plan to implement anything in that direction any time soon, I just have the bandwidth for that and fear that some minor steps in that direction wouldn't be all that useful in isolation.

There is already a package that provides Github-Org integration but I don't remember its name and it has been abandoned years ago. Still, you might want to locate and investigate that package and maybe adapt it to use forge as an intermediate layer.

**Reply by Thaodan** [reply]
This one maybe? 
https://github.com/lujun9972/org2issue

**Reply by tarsius** [reply]
No, the one I had in mind was ~~much~~ more featureful.

**Reply by tarsius** [reply]
Found it: https://orgmode.org/worg/org-contrib/gsoc2012/student-projects/org-sync/.

**Reply by tarsius** [reply]
> While I understand the appeal of "Org support", I don't plan to implement anything in that direction any time soon,

I still don't and am closing this now.

**Comment by Thaodan** [comment]
This is the other one I know of:
https://github.com/dakra/org-github/blob/master/org-github.el

**Comment by Thaodan** [comment]
Oh this one, there a new repository for it:
https://github.com/arbox/org-sync
